templates/01_types.tpl

```diff
@@ -4,6 +4,9 @@
 {{- $tableNameSingular := .Table.Name | singular | titleCase -}}
 var (
 	{{$varNameSingular}}Columns               = []string{{"{"}}{{.Table.Columns | columnNames | stringMap .StringFuncs.quoteWrap | join ", "}}{{"}"}}
+	{{if eq .DriverName "mssql" -}}
+	{{$varNameSingular}}ColumnsWithAuto = []string{{"{"}}{{.Table.Columns | filterColumnsByAuto true | columnNames | stringMap .StringFuncs.quoteWrap | join ","}}{{"}"}}
+	{{- end}}
```

Comment:
Fairly sure it should be `{{end -}}` here. You'll wind up with a newline for no reason if this template doesn't end up evaluating true I think.